pkg/nodeconfig/nodeconfig.go

```diff
@@ -505,11 +509,19 @@ func (nc *nodeConfig) Deconfigure() error {
 		return errors.Wrapf(err, "unable to drain node %s", nc.node.GetName())
 	}
 
+	if err := nc.Windows.DeconfigureWICD(); err != nil {
+		return errors.Wrap(err, "deconfiguring WICD failed")
+	}
+
 	// Revert the changes we've made to the instance by removing services and deleting all installed files
-	if err := nc.Windows.Deconfigure(); err != nil {
+	if err := nc.Windows.Deconfigure(preserveNode, nodeConfigCache.apiServerEndpoint, nodeConfigCache.credentials); err != nil {
 		return errors.Wrap(err, "error deconfiguring instance")
 	}
+	nc.log.Info("instance has been deconfigured", "node", nc.node.GetName())
```

Comment:
This has the possiblity of printing multiple times for a single instance if preserveNode is set and the RemovePatch fails and this is re-reconciled.